oauthproxy.go

```diff
@@ -563,7 +565,7 @@ func (p *OAuthProxy) GetRedirect(req *http.Request) (redirect string, err error)
 	}
 
 	redirect = req.Form.Get("rd")
-	if redirect == "" || !strings.HasPrefix(redirect, "/") || strings.HasPrefix(redirect, "//") {
+	if !p.IsValidRedirect(redirect) {
```

Comment:
@syscll Can you quickly double check the behaviour hasn't changed here with this, I don't think I'm changing the behaviour of having an empty rd parameter defaulting to the request path?